package.json

```diff
@@ -31,8 +31,8 @@
       "order-lines": "2.0 3.0",
       "orders": "11.0 12.0",
       "organizations.organizations": "1.0",
-      "pieces": "2.0",
-      "receiving": "1.0",
+      "pieces": "3.0",
+      "receiving": "2.0",
```

Comment:
Bumping an okapi interface version is akin to bumping a peer-dependency version; this is a breaking change. Please update the module version to 5.0.0 and call out this change in the CHANGELOG, e.g.
```
* *BREAKING* Bump minimum okapi interfaces for `pieces` (3.0) and `receiving` (2.0). Refs UIREC-316.
```

Reply:
Update the PR according to your comments.